Cdn_Plugin_Admin.php

```diff
@@ -191,6 +191,11 @@ public function w3tc_settings_general_boxarea_cdn() {
 			'optgroup' => $optgroup_push,
 		);
 
+		$engine_values['azuremi'] = array(
+			'label'    => \__( 'Microsoft Azure Storage (Managed Identity)', 'w3-total-cache' ),
```

Comment:
Microsoft Azure Storage with Managed Identity (Recommended) ?

Reply:
We cannot add (Recommended) suffix here because there are so many other options like S3, Google drive etc., It would make it look like that Azure Blob Storage is recommended among all the content distribution options. The change might get rejected.

Please do let me know what you think?